src/current/v24.1/cockroach-start.md

```diff
@@ -223,6 +227,57 @@ Field | Description
+
+#### Write Ahead Log (WAL) Failover
+
+{% include_cached new-in.html version="v24.1" %}
+
+When a CockroachDB [node]({% link {{ page.version.version }}/architecture/overview.md %}#node) is configured to run with [multiple stores](#store), you can mitigate some effects of [disk stalls]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#disk-stalls) by configuring the node to failover the store's [write-ahead log (WAL)]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) to another store's data directory.
+
+Failing over the WAL may allow some operations against a store to continue to complete despite temporary unavailability of the underlying storage. For example, if the node's primary store is stalled, and the node can't read or write from it, the node can still write to the WAL on another store. This can give the node a chance to eventually catch up once the disk stall has been resolved.
+
+When WAL failover is enabled, CockroachDB will take the the following actions:
+
+- At node startup, each store is assigned another store to be its failover destination.
+- CockroachDB will begin monitoring the latency of all WAL writes. If latency to the WAL exceeds the value of the [cluster setting `storage.wal_failover.unhealthy_op_threshold`]({% link {{page.version.version}}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold), the node will attempt to write WAL entries to a secondary store's volume.
+- CockroachDB will update the [store status endpoint]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#store-status-endpoint) at `/_status/stores` so you can monitor the store's status.
+
+{% include feature-phases/preview.md %}
+
+When this feature exits preview status and is generally available (GA), it will be an [Enterprise feature]({% link {{ page.version.version }}/enterprise-licensing.md %}).
+
+##### Enable WAL failover
+
+To enable WAL failover, you must take one of the following actions:
+
+- Pass [`--wal-failover=among-stores`](#flag-wal-failover) to `cockroach start`, or
+- Set the environment variable `COCKROACH_WAL_FAILOVER=among-stores` before starting the node.
+
+If you enable WAL failover, you must also update your [logging]({% link {{page.version.version}}/logging-overview.md %}) configuration as described below. This is necessary because although WAL failover alone is not enough to give a node time to recover from a [disk stall]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#disk-stalls) without the node crashing or becoming unstable. The disk unavailability will create instability via another vector: [file logging]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files).
```

Comment:
`This is necessary because although WAL failover alone is not enough to give a node time to recover from a [disk stall]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#disk-stalls) without the node crashing or becoming unstable. The disk unavailability will create instability via another vector: [file logging]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files).`

Sentence reads weird. Not a blocking nit but should be fixed before we push.

Reply:
Took a shot at a revision, PTAL!